src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  // Rozdzielamy ciąg na linie, usuwamy puste linie i linie zawierające tylko białe znaki
+  const lines = sourceString.split(';').filter(line => line.trim());
+
+  lines.forEach(line => {
```

Comment:
Use of `forEach` to modify the `stylesObject` is acceptable, but consider using more functional approaches like `reduce` for such transformations as it aligns better with the checklist item 5.